docs: add how to install customized nvidia driver (#887)
harvester/docs: docs/advanced/addons/nvidiadrivertoolkit.md

Once the addon is enabled, a nvidia-driver-toolkit daemonset is deployed to the cluster.

On pod startup, the ENTRYPOINT script will download the NVIDIA driver from the specified `Driver Location`. Install the driver and load the kernel drivers.

The `PCIDevices` addon can now leverage this addon to manage the lifecycle of the vGPU devices on nodes containing supported GPU [devices](../vgpusupport.md).

> ## Install Customized NVIDIA Driver

Reviewer: ```suggestion
## Install Different NVIDIA Driver Versions
```

_Available as of v1.7.0_

> Harvester introduces a new feature that allows users to install different NVIDIA driver versions on different nodes.
> 
> Before starting the nvidia-driver-toolkit, annotate the node where you want to install a specific driver version:

Reviewer: ```suggestion
NVIDIA driver versions can vary across cluster nodes. If you want to install a specific driver version on a node, you must annotate the node before starting the nvidia-driver-toolkit add-on.
```

> - `kubectl annotate nodes {node name} sriovgpu.harvesterhci.io/custom-driver=https://[driver location]`

Reviewer: ````suggestion
```
kubectl annotate nodes {node name} sriovgpu.harvesterhci.io/custom-driver=https://[driver location]
```
````

> Then, start the nvidia-driver-toolkit. It will install the specified driver version.
> 
> For users who have previously installed an NVIDIA driver version, you need to restart the pod to trigger the installation process again.

Reviewer: ```suggestion
The nvidia-driver-toolkit installs the specified driver version upon starting.

If an NVIDIA driver was previously installed, you must restart the pod to trigger the installation process again.
```
